res/layout-sw320dp/start.xml

```diff
@@ -30,6 +30,7 @@
         android:layout_width="fill_parent"
         android:layout_height="fill_parent"
         android:layout_marginTop="@dimen/lv_margin_top"
-        android:divider="@null"
-        />
+        android:divider="@drawable/lv_divider_inset_normal"
+        android:footerDividersEnabled="false"
```

Comment:
We can\`t do that because each listview has applied to a different divider, like `lv_divider_inset_normal`, `lv_divider_inset_small` and `lv_divider_inset_large`.